storefront/features/Header/Header.tsx

```diff
@@ -1,11 +1,18 @@
 import styled from "styled-components";
 import { Heading } from "../../components";
 import Navbar from "../../containers/Navbar/Navbar";
+import Image from 'next/image'
+import logo from '../images/logo.png'
 
 const Header = () => {
   return (
     <Wrapper>
-      <Heading textAlign="center">Aixmazone</Heading>
+      <Heading textAlign="center">
+      <Image
+              src={logo}
```

Comment:
Regarding accessibility, can't you add something that could help blind people that can't see images for example?